Makefile

```diff
@@ -229,3 +229,7 @@ coverage/report-xml: coverage/combine
 .PHONY: coverage/report
 coverage/report-html: coverage/combine
 	docker-compose $(COMPOSE_OPTS) run --rm -T backend bash -c "cd /single-cell-data-portal && coverage html --data-file=$(COVERAGE_DATA_FILE) -i --skip-empty"
+
+.PHONY: promote-staging-to-prod # set GITHUB_PAT env var OR call as "make GITHUB_PAT=<github_pat> promote-staging-to-prod"
```

Comment:
can we work this into the recipe so the user doesn't have to think about it?

Reply:
If it's simple, go for it, otherwise save it for a future improvement.